cli/src/pcluster/models/cluster.py

```diff
@@ -368,6 +369,12 @@ def _upload_artifacts(self):
                     key=self._get_default_template_key(),
                 )
 
+            # Fixme: the code doesn't work for awsbatch
+            AWSApi.instance().s3.put_object(
```

Comment:
let's add `self.scheduling.scheudler != "awsbatch"` for now and keep the fixme.

Reply:
Done